models/src/main/java/com/vimeo/networking2/annotations/Internal.kt

```diff
@@ -0,0 +1,9 @@
+package com.vimeo.networking2.annotations
+
+/**
+ * Annotation to describe certain models as internal.
```

Comment:
"models or properties."

Also, might be helpful to elaborate on what internal means... internal to the library or internal to vimeo? Obviously we know what they mean, but consumer might not understand. I'd suggest using the phrasing "...as being for internal Vimeo use only."